routes/curations.js

```diff
@@ -90,4 +98,16 @@ function setup(service) {
   return router
 }
 
+//synchronous version
+function readTextFile(file) {
+    var fs=require('fs');
```

Comment:
const should be used where value won't change.

Reply:
Thanks Alex. I agree. I have updated the code with your suggested edits and re-committed